packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -173,6 +173,86 @@
     "description": "Label read out by accessibility tools (TalkBack or VoiceOver) for a modal barrier to indicate that a tap dismisses the barrier. A modal barrier can for example be found behind a alert or popup to block user interaction with elements behind it."
   },
 
+  "dateSeparator": "/",
```

Comment:
For the `_en.arb` descriptions, could we specify if the label is going to be a visual element, a semantics label, or both? I recently had translators ask me about this with the Cupertino tabs semantics label and the translations for certain languages will be unique based on that context.

Reply:
I was just following the pattern I saw in the existing labels in the file. They are all visual elements unless described as an 'accessibility label' or 'accessibility announcement'.